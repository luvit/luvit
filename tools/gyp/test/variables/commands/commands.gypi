# Copyright (c) 2009 Google Inc. All rights reserved.
# Use of this source code is governed by a BSD-style license that can be
# found in the LICENSE file.

# This file is included from commands.gyp to test evaluation order of includes.
{
  'variables': {
    'included_variable': 'XYZ',
  },
  'targets': [
    {
      'target_name': 'dummy',
      'type': 'none',
    },
  ],
}
