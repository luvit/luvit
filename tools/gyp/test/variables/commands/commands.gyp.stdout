GENERAL:__init__.py:356:main running with these options:
GENERAL:__init__.py:363:main   check: None
GENERAL:__init__.py:363:main   circular_check: True
GENERAL:__init__.py:363:main   debug: ['variables', 'general']
GENERAL:__init__.py:363:main   defines: None
GENERAL:__init__.py:361:main   depth: '.'
GENERAL:__init__.py:363:main   formats: ['gypd']
GENERAL:__init__.py:363:main   generator_flags: []
GENERAL:__init__.py:363:main   generator_output: None
GENERAL:__init__.py:363:main   includes: None
GENERAL:__init__.py:363:main   msvs_version: None
GENERAL:__init__.py:361:main   suffix: ''
GENERAL:__init__.py:363:main   toplevel_dir: None
GENERAL:__init__.py:363:main   use_environment: True
GENERAL:__init__.py:417:main cmdline_default_variables: {}
GENERAL:__init__.py:443:main generator_flags: {}
VARIABLES:input.py:785:ExpandVariables Expanding '0' to 0
VARIABLES:input.py:785:ExpandVariables Expanding '11 ' to '11 '
VARIABLES:input.py:785:ExpandVariables Expanding '+14' to '+14'
VARIABLES:input.py:785:ExpandVariables Expanding '-15' to -15
VARIABLES:input.py:785:ExpandVariables Expanding ' 10' to ' 10'
VARIABLES:input.py:546:ExpandVariables Matches: {'content': 'letters_list', 'is_array': '', 'replace': '<(letters_list)', 'type': '<', 'command_string': None}
VARIABLES:input.py:785:ExpandVariables Expanding 'letters_list' to 'letters_list'
VARIABLES:input.py:767:ExpandVariables Found output 'ABCDEFG', recursing.
VARIABLES:input.py:785:ExpandVariables Expanding 'ABCDEFG' to 'ABCDEFG'
VARIABLES:input.py:785:ExpandVariables Expanding '<(letters_list)EFG' to 'ABCDEFG'
VARIABLES:input.py:785:ExpandVariables Expanding '012' to '012'
VARIABLES:input.py:546:ExpandVariables Matches: {'content': 'other_letters', 'is_array': '', 'replace': '<(other_letters)', 'type': '<', 'command_string': None}
VARIABLES:input.py:785:ExpandVariables Expanding 'other_letters' to 'other_letters'
VARIABLES:input.py:767:ExpandVariables Found output '<(letters_list)EFGHIJK', recursing.
VARIABLES:input.py:546:ExpandVariables Matches: {'content': 'letters_list', 'is_array': '', 'replace': '<(letters_list)', 'type': '<', 'command_string': None}
VARIABLES:input.py:785:ExpandVariables Expanding 'letters_list' to 'letters_list'
VARIABLES:input.py:767:ExpandVariables Found output 'ABCDEFGHIJK', recursing.
VARIABLES:input.py:785:ExpandVariables Expanding 'ABCDEFGHIJK' to 'ABCDEFGHIJK'
VARIABLES:input.py:785:ExpandVariables Expanding '<(letters_list)EFGHIJK' to 'ABCDEFGHIJK'
VARIABLES:input.py:785:ExpandVariables Expanding '<(other_letters)HIJK' to 'ABCDEFGHIJK'
VARIABLES:input.py:785:ExpandVariables Expanding 'XYZ' to 'XYZ'
VARIABLES:input.py:785:ExpandVariables Expanding 'ABCD' to 'ABCD'
VARIABLES:input.py:785:ExpandVariables Expanding '13.0' to '13.0'
VARIABLES:input.py:785:ExpandVariables Expanding 'import math; print math.pi' to 'import math; print math.pi'
VARIABLES:input.py:546:ExpandVariables Matches: {'content': 'included_variable', 'is_array': '', 'replace': '<(included_variable)', 'type': '<', 'command_string': None}
VARIABLES:input.py:785:ExpandVariables Expanding 'included_variable' to 'included_variable'
VARIABLES:input.py:767:ExpandVariables Found output 'XYZ', recursing.
VARIABLES:input.py:785:ExpandVariables Expanding 'XYZ' to 'XYZ'
VARIABLES:input.py:785:ExpandVariables Expanding '<(included_variable)' to 'XYZ'
VARIABLES:input.py:785:ExpandVariables Expanding '6' to 6
VARIABLES:input.py:546:ExpandVariables Matches: {'content': 'included_variable', 'is_array': '', 'replace': '<(included_variable)', 'type': '<', 'command_string': None}
VARIABLES:input.py:785:ExpandVariables Expanding 'included_variable' to 'included_variable'
VARIABLES:input.py:767:ExpandVariables Found output 'XYZ', recursing.
VARIABLES:input.py:785:ExpandVariables Expanding 'XYZ' to 'XYZ'
VARIABLES:input.py:785:ExpandVariables Expanding '<(included_variable)' to 'XYZ'
VARIABLES:input.py:546:ExpandVariables Matches: {'content': 'third_letters', 'is_array': '', 'replace': '<(third_letters)', 'type': '<', 'command_string': None}
VARIABLES:input.py:785:ExpandVariables Expanding 'third_letters' to 'third_letters'
VARIABLES:input.py:767:ExpandVariables Found output '<(other_letters)HIJK', recursing.
VARIABLES:input.py:546:ExpandVariables Matches: {'content': 'other_letters', 'is_array': '', 'replace': '<(other_letters)', 'type': '<', 'command_string': None}
VARIABLES:input.py:785:ExpandVariables Expanding 'other_letters' to 'other_letters'
VARIABLES:input.py:767:ExpandVariables Found output '<(letters_list)EFGHIJK', recursing.
VARIABLES:input.py:546:ExpandVariables Matches: {'content': 'letters_list', 'is_array': '', 'replace': '<(letters_list)', 'type': '<', 'command_string': None}
VARIABLES:input.py:785:ExpandVariables Expanding 'letters_list' to 'letters_list'
VARIABLES:input.py:767:ExpandVariables Found output 'ABCDEFGHIJK', recursing.
VARIABLES:input.py:785:ExpandVariables Expanding 'ABCDEFGHIJK' to 'ABCDEFGHIJK'
VARIABLES:input.py:785:ExpandVariables Expanding '<(letters_list)EFGHIJK' to 'ABCDEFGHIJK'
VARIABLES:input.py:785:ExpandVariables Expanding '<(other_letters)HIJK' to 'ABCDEFGHIJK'
VARIABLES:input.py:785:ExpandVariables Expanding '<(third_letters)' to 'ABCDEFGHIJK'
VARIABLES:input.py:785:ExpandVariables Expanding '8' to 8
VARIABLES:input.py:785:ExpandVariables Expanding '.' to '.'
VARIABLES:input.py:785:ExpandVariables Expanding 'letters_' to 'letters_'
VARIABLES:input.py:546:ExpandVariables Matches: {'content': '<!(python -c "print \'letters_list\'"', 'is_array': '', 'replace': '<(<!(python -c "print \'letters_list\'")', 'type': '<', 'command_string': None}
VARIABLES:input.py:546:ExpandVariables Matches: {'content': 'python -c "print \'letters_list\'"', 'is_array': '', 'replace': '<!(python -c "print \'letters_list\'")', 'type': '<!', 'command_string': None}
VARIABLES:input.py:785:ExpandVariables Expanding 'python -c "print \'letters_list\'"' to 'python -c "print \'letters_list\'"'
VARIABLES:input.py:660:ExpandVariables Executing command 'python -c "print 'letters_list'"' in directory 'None'
VARIABLES:input.py:767:ExpandVariables Found output 'letters_list', recursing.
VARIABLES:input.py:785:ExpandVariables Expanding 'letters_list' to 'letters_list'
VARIABLES:input.py:785:ExpandVariables Expanding '<!(python -c "print \'letters_list\'")' to 'letters_list'
VARIABLES:input.py:767:ExpandVariables Found output 'ABCD', recursing.
VARIABLES:input.py:785:ExpandVariables Expanding 'ABCD' to 'ABCD'
VARIABLES:input.py:785:ExpandVariables Expanding '<(<!(python -c "print \'letters_list\'"))' to 'ABCD'
VARIABLES:input.py:546:ExpandVariables Matches: {'content': 'check_int', 'is_array': '', 'replace': '<(check_int)', 'type': '<', 'command_string': None}
VARIABLES:input.py:785:ExpandVariables Expanding 'check_int' to 'check_int'
VARIABLES:input.py:767:ExpandVariables Found output '5', recursing.
VARIABLES:input.py:785:ExpandVariables Expanding '5' to 5
VARIABLES:input.py:785:ExpandVariables Expanding '<(check_int)' to 5
VARIABLES:input.py:785:ExpandVariables Expanding 'list' to 'list'
VARIABLES:input.py:546:ExpandVariables Matches: {'content': '"python", "-c", "<(pi', 'is_array': '[', 'replace': '<!(["python", "-c", "<(pi)', 'type': '<!', 'command_string': None}
VARIABLES:input.py:546:ExpandVariables Matches: {'content': 'pi', 'is_array': '', 'replace': '<(pi)', 'type': '<', 'command_string': None}
VARIABLES:input.py:785:ExpandVariables Expanding 'pi' to 'pi'
VARIABLES:input.py:767:ExpandVariables Found output '["python", "-c", "import math; print math.pi"]', recursing.
VARIABLES:input.py:785:ExpandVariables Expanding '["python", "-c", "import math; print math.pi"]' to '["python", "-c", "import math; print math.pi"]'
VARIABLES:input.py:785:ExpandVariables Expanding '["python", "-c", "<(pi)"]' to '["python", "-c", "import math; print math.pi"]'
VARIABLES:input.py:660:ExpandVariables Executing command '['python', '-c', 'import math; print math.pi']' in directory 'None'
VARIABLES:input.py:767:ExpandVariables Found output '3.14159265359', recursing.
VARIABLES:input.py:785:ExpandVariables Expanding '3.14159265359' to '3.14159265359'
VARIABLES:input.py:785:ExpandVariables Expanding '<!(["python", "-c", "<(pi)"])' to '3.14159265359'
VARIABLES:input.py:546:ExpandVariables Matches: {'content': 'python -c "print \'<(letters_list', 'is_array': '', 'replace': '<!(python -c "print \'<(letters_list)', 'type': '<!', 'command_string': None}
VARIABLES:input.py:546:ExpandVariables Matches: {'content': 'letters_list', 'is_array': '', 'replace': '<(letters_list)', 'type': '<', 'command_string': None}
VARIABLES:input.py:785:ExpandVariables Expanding 'letters_list' to 'letters_list'
VARIABLES:input.py:767:ExpandVariables Found output 'python -c "print \'ABCD\'"', recursing.
VARIABLES:input.py:785:ExpandVariables Expanding 'python -c "print \'ABCD\'"' to 'python -c "print \'ABCD\'"'
VARIABLES:input.py:785:ExpandVariables Expanding 'python -c "print \'<(letters_list)\'"' to 'python -c "print \'ABCD\'"'
VARIABLES:input.py:660:ExpandVariables Executing command 'python -c "print 'ABCD'"' in directory 'None'
VARIABLES:input.py:767:ExpandVariables Found output 'ABCD', recursing.
VARIABLES:input.py:785:ExpandVariables Expanding 'ABCD' to 'ABCD'
VARIABLES:input.py:785:ExpandVariables Expanding '<!(python -c "print \'<(letters_list)\'")' to 'ABCD'
VARIABLES:input.py:546:ExpandVariables Matches: {'content': 'letters_list', 'is_array': '', 'replace': '<(letters_list)', 'type': '<', 'command_string': None}
VARIABLES:input.py:785:ExpandVariables Expanding 'letters_list' to 'letters_list'
VARIABLES:input.py:546:ExpandVariables Matches: {'content': 'python -c "print \'<!(python -c "<(pi', 'is_array': '', 'replace': '<!(python -c "print \'<!(python -c "<(pi)', 'type': '<!', 'command_string': None}
VARIABLES:input.py:546:ExpandVariables Matches: {'content': 'python -c "<(pi', 'is_array': '', 'replace': '<!(python -c "<(pi)', 'type': '<!', 'command_string': None}
VARIABLES:input.py:546:ExpandVariables Matches: {'content': 'pi', 'is_array': '', 'replace': '<(pi)', 'type': '<', 'command_string': None}
VARIABLES:input.py:785:ExpandVariables Expanding 'pi' to 'pi'
VARIABLES:input.py:767:ExpandVariables Found output 'python -c "import math; print math.pi"', recursing.
VARIABLES:input.py:785:ExpandVariables Expanding 'python -c "import math; print math.pi"' to 'python -c "import math; print math.pi"'
VARIABLES:input.py:785:ExpandVariables Expanding 'python -c "<(pi)"' to 'python -c "import math; print math.pi"'
VARIABLES:input.py:660:ExpandVariables Executing command 'python -c "import math; print math.pi"' in directory 'None'
VARIABLES:input.py:767:ExpandVariables Found output 'python -c "print \'3.14159265359 ABCD\'"', recursing.
VARIABLES:input.py:785:ExpandVariables Expanding 'python -c "print \'3.14159265359 ABCD\'"' to 'python -c "print \'3.14159265359 ABCD\'"'
VARIABLES:input.py:785:ExpandVariables Expanding 'python -c "print \'<!(python -c "<(pi)") ABCD\'"' to 'python -c "print \'3.14159265359 ABCD\'"'
VARIABLES:input.py:660:ExpandVariables Executing command 'python -c "print '3.14159265359 ABCD'"' in directory 'None'
VARIABLES:input.py:767:ExpandVariables Found output '3.14159265359 ABCD', recursing.
VARIABLES:input.py:785:ExpandVariables Expanding '3.14159265359 ABCD' to '3.14159265359 ABCD'
VARIABLES:input.py:785:ExpandVariables Expanding '<!(python -c "print \'<!(python -c "<(pi)") <(letters_list)\'")' to '3.14159265359 ABCD'
VARIABLES:input.py:546:ExpandVariables Matches: {'content': 'check_str_int', 'is_array': '', 'replace': '<(check_str_int)', 'type': '<', 'command_string': None}
VARIABLES:input.py:785:ExpandVariables Expanding 'check_str_int' to 'check_str_int'
VARIABLES:input.py:767:ExpandVariables Found output '6', recursing.
VARIABLES:input.py:785:ExpandVariables Expanding '6' to 6
VARIABLES:input.py:785:ExpandVariables Expanding '<(check_str_int)' to 6
VARIABLES:input.py:546:ExpandVariables Matches: {'content': 'check_int', 'is_array': '', 'replace': '<(check_int)', 'type': '<', 'command_string': None}
VARIABLES:input.py:785:ExpandVariables Expanding 'check_int' to 'check_int'
VARIABLES:input.py:767:ExpandVariables Found output '5blah', recursing.
VARIABLES:input.py:785:ExpandVariables Expanding '5blah' to '5blah'
VARIABLES:input.py:785:ExpandVariables Expanding '<(check_int)blah' to '5blah'
VARIABLES:input.py:546:ExpandVariables Matches: {'content': 'not_int_4', 'is_array': '', 'replace': '<(not_int_4)', 'type': '<', 'command_string': None}
VARIABLES:input.py:785:ExpandVariables Expanding 'not_int_4' to 'not_int_4'
VARIABLES:input.py:767:ExpandVariables Found output '13.0', recursing.
VARIABLES:input.py:785:ExpandVariables Expanding '13.0' to '13.0'
VARIABLES:input.py:785:ExpandVariables Expanding '<(not_int_4)' to '13.0'
VARIABLES:input.py:546:ExpandVariables Matches: {'content': 'not_int_3', 'is_array': '', 'replace': '<(not_int_3)', 'type': '<', 'command_string': None}
VARIABLES:input.py:785:ExpandVariables Expanding 'not_int_3' to 'not_int_3'
VARIABLES:input.py:767:ExpandVariables Found output '012', recursing.
VARIABLES:input.py:785:ExpandVariables Expanding '012' to '012'
VARIABLES:input.py:785:ExpandVariables Expanding '<(not_int_3)' to '012'
VARIABLES:input.py:546:ExpandVariables Matches: {'content': 'negative_int', 'is_array': '', 'replace': '<(negative_int)', 'type': '<', 'command_string': None}
VARIABLES:input.py:785:ExpandVariables Expanding 'negative_int' to 'negative_int'
VARIABLES:input.py:767:ExpandVariables Found output '-15', recursing.
VARIABLES:input.py:785:ExpandVariables Expanding '-15' to -15
VARIABLES:input.py:785:ExpandVariables Expanding '<(negative_int)' to -15
VARIABLES:input.py:546:ExpandVariables Matches: {'content': 'not_int_5', 'is_array': '', 'replace': '<(not_int_5)', 'type': '<', 'command_string': None}
VARIABLES:input.py:785:ExpandVariables Expanding 'not_int_5' to 'not_int_5'
VARIABLES:input.py:767:ExpandVariables Found output '+14', recursing.
VARIABLES:input.py:785:ExpandVariables Expanding '+14' to '+14'
VARIABLES:input.py:785:ExpandVariables Expanding '<(not_int_5)' to '+14'
VARIABLES:input.py:546:ExpandVariables Matches: {'content': 'check_list_int', 'is_array': '', 'replace': '<(check_list_int)', 'type': '<', 'command_string': None}
VARIABLES:input.py:785:ExpandVariables Expanding 'check_list_int' to 'check_list_int'
VARIABLES:input.py:767:ExpandVariables Found output '7 8 9', recursing.
VARIABLES:input.py:785:ExpandVariables Expanding '7 8 9' to '7 8 9'
VARIABLES:input.py:785:ExpandVariables Expanding '<(check_list_int)' to '7 8 9'
VARIABLES:input.py:546:ExpandVariables Matches: {'content': 'not_int_2', 'is_array': '', 'replace': '<(not_int_2)', 'type': '<', 'command_string': None}
VARIABLES:input.py:785:ExpandVariables Expanding 'not_int_2' to 'not_int_2'
VARIABLES:input.py:767:ExpandVariables Found output '11 ', recursing.
VARIABLES:input.py:785:ExpandVariables Expanding '11 ' to '11 '
VARIABLES:input.py:785:ExpandVariables Expanding '<(not_int_2)' to '11 '
VARIABLES:input.py:546:ExpandVariables Matches: {'content': 'not_int_1', 'is_array': '', 'replace': '<(not_int_1)', 'type': '<', 'command_string': None}
VARIABLES:input.py:785:ExpandVariables Expanding 'not_int_1' to 'not_int_1'
VARIABLES:input.py:767:ExpandVariables Found output ' 10', recursing.
VARIABLES:input.py:785:ExpandVariables Expanding ' 10' to ' 10'
VARIABLES:input.py:785:ExpandVariables Expanding '<(not_int_1)' to ' 10'
VARIABLES:input.py:546:ExpandVariables Matches: {'content': 'zero_int', 'is_array': '', 'replace': '<(zero_int)', 'type': '<', 'command_string': None}
VARIABLES:input.py:785:ExpandVariables Expanding 'zero_int' to 'zero_int'
VARIABLES:input.py:767:ExpandVariables Found output '0', recursing.
VARIABLES:input.py:785:ExpandVariables Expanding '0' to 0
VARIABLES:input.py:785:ExpandVariables Expanding '<(zero_int)' to 0
VARIABLES:input.py:546:ExpandVariables Matches: {'content': 'check_list_int', 'is_array': '', 'replace': '<@(check_list_int)', 'type': '<@', 'command_string': None}
VARIABLES:input.py:785:ExpandVariables Expanding 'check_list_int' to 'check_list_int'
VARIABLES:input.py:767:ExpandVariables Found output [7, 8, 9], recursing.
VARIABLES:input.py:785:ExpandVariables Expanding 7 to 7
VARIABLES:input.py:785:ExpandVariables Expanding 8 to 8
VARIABLES:input.py:785:ExpandVariables Expanding 9 to 9
VARIABLES:input.py:785:ExpandVariables Expanding '<@(check_list_int)' to [7, 8, 9]
VARIABLES:input.py:785:ExpandVariables Expanding 'foo' to 'foo'
VARIABLES:input.py:785:ExpandVariables Expanding 'target' to 'target'
VARIABLES:input.py:785:ExpandVariables Expanding 'none' to 'none'
VARIABLES:input.py:546:ExpandVariables Matches: {'content': 'var6', 'is_array': '', 'replace': '<(var6)', 'type': '<', 'command_string': None}
VARIABLES:input.py:785:ExpandVariables Expanding 'var6' to 'var6'
VARIABLES:input.py:546:ExpandVariables Matches: {'content': 'echo <(var5', 'is_array': '', 'replace': '<!(echo <(var5)', 'type': '<!', 'command_string': None}
VARIABLES:input.py:546:ExpandVariables Matches: {'content': 'var5', 'is_array': '', 'replace': '<(var5)', 'type': '<', 'command_string': None}
VARIABLES:input.py:785:ExpandVariables Expanding 'var5' to 'var5'
VARIABLES:input.py:767:ExpandVariables Found output 'echo letters_list', recursing.
VARIABLES:input.py:785:ExpandVariables Expanding 'echo letters_list' to 'echo letters_list'
VARIABLES:input.py:785:ExpandVariables Expanding 'echo <(var5)list' to 'echo letters_list'
VARIABLES:input.py:660:ExpandVariables Executing command 'echo letters_list' in directory 'None'
VARIABLES:input.py:767:ExpandVariables Found output 'letters_list', recursing.
VARIABLES:input.py:785:ExpandVariables Expanding 'letters_list' to 'letters_list'
VARIABLES:input.py:785:ExpandVariables Expanding '<!(echo <(var5)<(var6))' to 'letters_list'
VARIABLES:input.py:785:ExpandVariables Expanding 'test_action' to 'test_action'
VARIABLES:input.py:785:ExpandVariables Expanding 'echo' to 'echo'
VARIABLES:input.py:546:ExpandVariables Matches: {'content': '_inputs', 'is_array': '', 'replace': '<(_inputs)', 'type': '<', 'command_string': None}
VARIABLES:input.py:785:ExpandVariables Expanding '_inputs' to '_inputs'
VARIABLES:input.py:546:ExpandVariables Matches: {'content': 'var2', 'is_array': '', 'replace': '<(var2)', 'type': '<', 'command_string': None}
VARIABLES:input.py:785:ExpandVariables Expanding 'var2' to 'var2'
VARIABLES:input.py:767:ExpandVariables Found output '3.14159265359 ABCD', recursing.
VARIABLES:input.py:785:ExpandVariables Expanding '3.14159265359 ABCD' to '3.14159265359 ABCD'
VARIABLES:input.py:785:ExpandVariables Expanding '<(var2)' to '3.14159265359 ABCD'
VARIABLES:input.py:767:ExpandVariables Found output '"3.14159265359 ABCD"', recursing.
VARIABLES:input.py:785:ExpandVariables Expanding '"3.14159265359 ABCD"' to '"3.14159265359 ABCD"'
VARIABLES:input.py:785:ExpandVariables Expanding '<(_inputs)' to '"3.14159265359 ABCD"'
VARIABLES:input.py:546:ExpandVariables Matches: {'content': '_outputs', 'is_array': '', 'replace': '<(_outputs)', 'type': '<', 'command_string': None}
VARIABLES:input.py:785:ExpandVariables Expanding '_outputs' to '_outputs'
VARIABLES:input.py:546:ExpandVariables Matches: {'content': 'var4', 'is_array': '', 'replace': '<(var4)', 'type': '<', 'command_string': None}
VARIABLES:input.py:785:ExpandVariables Expanding 'var4' to 'var4'
VARIABLES:input.py:767:ExpandVariables Found output 'ABCD', recursing.
VARIABLES:input.py:785:ExpandVariables Expanding 'ABCD' to 'ABCD'
VARIABLES:input.py:785:ExpandVariables Expanding '<(var4)' to 'ABCD'
VARIABLES:input.py:546:ExpandVariables Matches: {'content': 'var7', 'is_array': '', 'replace': '<(var7)', 'type': '<', 'command_string': None}
VARIABLES:input.py:785:ExpandVariables Expanding 'var7' to 'var7'
VARIABLES:input.py:767:ExpandVariables Found output 'letters_list', recursing.
VARIABLES:input.py:785:ExpandVariables Expanding 'letters_list' to 'letters_list'
VARIABLES:input.py:785:ExpandVariables Expanding '<(var7)' to 'letters_list'
VARIABLES:input.py:767:ExpandVariables Found output 'ABCD letters_list', recursing.
VARIABLES:input.py:785:ExpandVariables Expanding 'ABCD letters_list' to 'ABCD letters_list'
VARIABLES:input.py:785:ExpandVariables Expanding '<(_outputs)' to 'ABCD letters_list'
VARIABLES:input.py:785:ExpandVariables Expanding '3.14159265359 ABCD' to '3.14159265359 ABCD'
VARIABLES:input.py:785:ExpandVariables Expanding 'ABCD' to 'ABCD'
VARIABLES:input.py:785:ExpandVariables Expanding 'letters_list' to 'letters_list'
VARIABLES:input.py:785:ExpandVariables Expanding 'dummy' to 'dummy'
VARIABLES:input.py:785:ExpandVariables Expanding 'target' to 'target'
VARIABLES:input.py:785:ExpandVariables Expanding 'none' to 'none'
VARIABLES:input.py:785:ExpandVariables Expanding 'commands.gyp' to 'commands.gyp'
VARIABLES:input.py:785:ExpandVariables Expanding 'commands.gypi' to 'commands.gypi'
VARIABLES:input.py:785:ExpandVariables Expanding 'dummy' to 'dummy'
VARIABLES:input.py:785:ExpandVariables Expanding 'target' to 'target'
VARIABLES:input.py:785:ExpandVariables Expanding 'none' to 'none'
VARIABLES:input.py:785:ExpandVariables Expanding 'letters_' to 'letters_'
VARIABLES:input.py:785:ExpandVariables Expanding 'ABCD' to 'ABCD'
VARIABLES:input.py:785:ExpandVariables Expanding 'list' to 'list'
VARIABLES:input.py:785:ExpandVariables Expanding '3.14159265359' to '3.14159265359'
VARIABLES:input.py:785:ExpandVariables Expanding 'ABCD' to 'ABCD'
VARIABLES:input.py:785:ExpandVariables Expanding '3.14159265359 ABCD' to '3.14159265359 ABCD'
VARIABLES:input.py:785:ExpandVariables Expanding '5blah' to '5blah'
VARIABLES:input.py:785:ExpandVariables Expanding '13.0' to '13.0'
VARIABLES:input.py:785:ExpandVariables Expanding '012' to '012'
VARIABLES:input.py:785:ExpandVariables Expanding '+14' to '+14'
VARIABLES:input.py:785:ExpandVariables Expanding '7 8 9' to '7 8 9'
VARIABLES:input.py:785:ExpandVariables Expanding '11 ' to '11 '
VARIABLES:input.py:785:ExpandVariables Expanding ' 10' to ' 10'
VARIABLES:input.py:785:ExpandVariables Expanding 'foo' to 'foo'
VARIABLES:input.py:785:ExpandVariables Expanding 'target' to 'target'
VARIABLES:input.py:785:ExpandVariables Expanding 'none' to 'none'
VARIABLES:input.py:785:ExpandVariables Expanding 'letters_list' to 'letters_list'
VARIABLES:input.py:785:ExpandVariables Expanding 'test_action' to 'test_action'
VARIABLES:input.py:785:ExpandVariables Expanding 'echo' to 'echo'
VARIABLES:input.py:785:ExpandVariables Expanding '"3.14159265359 ABCD"' to '"3.14159265359 ABCD"'
VARIABLES:input.py:785:ExpandVariables Expanding 'ABCD letters_list' to 'ABCD letters_list'
VARIABLES:input.py:785:ExpandVariables Expanding '3.14159265359 ABCD' to '3.14159265359 ABCD'
VARIABLES:input.py:785:ExpandVariables Expanding 'ABCD' to 'ABCD'
VARIABLES:input.py:785:ExpandVariables Expanding 'letters_list' to 'letters_list'
